app/controllers/items_controller.rb

```diff
@@ -1,5 +1,7 @@
 class ItemsController < ApplicationController
-
+  
+  helper_method :current_user 
```

Comment:
`current_user` should already be defined by devise. Try to figure out why it's not working and let me know if you need help debugging.